src/test/java/monobot/util/StorageTest.java

```diff
@@ -0,0 +1,95 @@
+package monobot.util;
+
+import monobot.exception.MonoBotException;
+import monobot.task.Deadline;
+import monobot.task.Event;
+import monobot.task.Task;
+import monobot.task.Todo;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Should the imported Assertion classes be listed explicitly instead of with a wildcard?